.gitignore

```diff
@@ -3,3 +3,6 @@ _site/
 
 # Files generated by RubyGems #
 docs/Gemfile.lock
```

Comment:
Ge@mvogelgesang can you update the .gitignore file:
docs/Gemfile.lock --> Gemfile.lock

I think we want to continue to exclude the Gemfile stuff from the repo, because it is causing me some issues on my side.